assets/webview/pages/table/index.js

```diff
@@ -157,16 +159,16 @@ VuePage(function(page) {
       },
       undo() {
         page.prompt.warn(
-          '撤销全部？',
-          '您可以使用 ctrl-z(windows) 或 cmd-z(macos) 来撤销一小步', 
+          page.state.l10n.allUndo,
+          page.state.l10n.forUndo, 
```

Comment:
- Undo all changes?
- You can use Ctrl+z(Windows/Linux) or Cmd+z(MacOS) to undo one step